src/nft/components/details/AssetActivity.tsx

```diff
@@ -84,75 +84,120 @@ const ActivityContainer = styled.div`
   ${ScrollBarStyles}
 `
 
-const AssetActivity = ({ eventsData }: { eventsData: ActivityEventResponse | undefined }) => {
+const LoadingCell = styled(LoadingBubble)`
+  height: 20px;
+  width: 80px;
+`
+
+enum AssetActivityColumnHeader {
+  Event = 'Event',
+  Price = 'Price',
+  By = 'By',
+  To = 'To',
+  Time = 'Time',
+}
```

Comment:
nit: Nice, to get started on internationalization we could wrap this text in `Trans` components